sopel/coretasks.py

```diff
@@ -347,6 +347,13 @@ def parse_reply_myinfo(bot, trigger):
     # the trailing parameters (mode types) should be read from ISUPPORT
     bot._myinfo = MyInfo(*trigger.args[0:3])
 
+    LOGGER.info(
```

Comment:
This just seems like a debug thing.

```suggestion
    LOGGER.debug(
```

Reply:
It's a one-time occurrence, and it provides an information that can be useful in some rare case. `DEBUG` is really to "debug-only" as it spams information. In that case, I believe the tradeof is acceptable: an almost debug information that isn't spammed can be an INFO.

Reply:
Fair enough.